src/Status/StatusCountTable/StatusCountTable.types.ts

```diff
@@ -0,0 +1,16 @@
+import { SxProps, Theme } from "@mui/material";
+
+export type StatusCountTableProps = {
+  /**
+   * Title of the table
+   */
+  title: string;
+  /**
+   * Count is an object which contains objects with key status name and status count
+   */
+  count: { [status: string]: number };
+  /**
+   * Optional style properties for the table element
+   */
+  tableSx?: SxProps<Theme>;
```

Comment:
What is the use case for this prop? We didn't propose it in the planning.

Reply:
I was suggesting that we may need to reuse this component on another place and change the style.

Reply:
We prefer not to expose sx from components as they potentially hide real use cases that we should support. If we don't have a concrete use case now then lets remove for now.